packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.tsx

```diff
@@ -571,6 +571,10 @@ export class FieldEditor {
       }
     }
 
+    if (this.dataProvider.name === 'status') {
```

Comment:
Checking for hardcode name like "status" will cause issues for other products. Can you add a config value in form-mapper.json which will check if the form builder supports toggle please. Add the name of the filed also in the same config

Reply:
updated